3.0/README.md

```diff
@@ -154,6 +154,55 @@ An array of tile endpoints. {z}, {x} and {y}, if present, are replaced with the
 ```
 
 ## 3.15 `vector_layers`
+
+REQUIRED if describing vector tiles.
```

Comment:
We're saying this is required, but how do we effectively cover non vector-tile cases that cannot be described in this format?